ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/wms_detail.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Map configuration via Web Map Service(WMS)</title>
+</head>
+<body>
+	<h1>WMS Configuration</h1>
+	<p>
+		The <i>WMS</i> provides functionality to add map using service URL. 
+	</p>
+	<p>
+		Here is the explanation of how to add, configure and remove <i>WMS</i> 
+		map in <i>Hale</i> studio. To work with WMS maps, first open Map perspective 
+		using Open Perspective tool menu reside in the right corner of the studio.
+    </p>
+    
+    <div>
+		<img src="../images/map_perspective_toolbar.png"
+			title="Map perspective tool menu" class="fit">
+		<img src="../images/map_perspective_open.png"
+			title="Map perspective tool menu" class="fit" style="margin-left:20px" >
+	</div>
+    
+    <p>
+    	To add the WMS maps in to Map perspective, click on the <i>View Menu</i> 
+    	button and then heads to <i>WMS maps</i>&#8594;<i>Add</i>. 
+	</p>
+	
+	<div>
+		<img src="../images/wms_add.png"
+			title="Add WMS map to Map perspective" class="fit">
+	</div>
+	
+	
+	<p>
+		By clicking on <i>Add</i> button, a dialog will be appeared, which asks 
+		for the <i>Name</i> and <i>Service URL</i>. A user have to insert a unique 
+		name and URL from where map tiles will be fetched.
+	</p>
+	<p>
+		Please note here name must be unique from any other <i>WMS</i> map configurations 
+		and URL should be valid to fetch capabilities. To go to the next configuration 
+		page, click on <i>Next</i> button in the dialog or click <i>Finish</i> directly 
+		to skip next configuration pages.
+	</p>
+	<div>
+		<img src="../images/wms_basic.png"
+			title="Basic configuration of WMS maps" class="fit">
+	</div>
+	<p>
+		On the next page, select <i>Spatial reference system</i> supported by the
+		<i>WMS</i>. expand the combo and select the respective <i>SRS</i>. To go to 
+		the next configuration page, click on <i>Next</i> button in the dialog or 
+		click <i>Finish</i> to skip next configuration pages and complete
+		configuration.
+	</p>
+	<div>
+		<img src="../images/wms_srs.png"
+			title="SRS configuration of WMS maps" class="fit">
+	</div>
+
+	<p>
+		On this page, select <i>Layers</i> supported by the <i>WMS</i>. Click on
```

Comment:
On this page, select the <i>Layers</i> supported by the <i>WMS</i> that you want to display in the Map view.